### ISSUE TITLE
AmazonMWSClient not defined?

### ISSUE THREAD

**Issue by danfinlay** [eibbors/mws-js#2]
I'm using your super simple example to start, but I'm getting:

```
ReferenceError: AmazonMwsClient is not defined
```

Any tips on getting around this?  I've never tried to construct a prototype defined within a module before, so I don't know how to debug this.

**Comment by mfelix** [comment]
heya @flyswatter, i'm new to node and the problem described here is what i'm stuck on. do you mind briefly explaining how you overcame this issue?

**Comment by danfinlay** [comment]
Yeah this problem was surprisingly easy to get around, and looks like the latest version makes it even easier.  Basically the README is wrong.  The basic example should look like this:

```
var mws = require('mws'),
client = new mws.Client('accessKeyId', 'secretAccessKey', 'merchantId', {});
```

That's because `AmazonMWSClient` is a function internal to the `mws` module, and it exposes it on the Client key [here](https://github.com/eibbors/mws-js/blob/master/lib/mws.js#L377).

You're welcome!  I'll push this fix to eibbors.

**Comment by mfelix** [comment]
right on, thanks for the tip + + +
